### DISCUSSION TITLE
可否提供一个默认随机播放的选项

### DISCUSSION THREAD

**Discussion by FM7077** [gitbobobo/StreamMusic#384] [General]
进入专辑或歌单点击播放, 默认是顺序播放, 可否提供一个设置, 可改为默认随机播放, 像 spotify 那样
![image](https://github.com/gitbobobo/StreamMusic/assets/41443707/24e0644a-ce48-4d71-9d26-022b935aecc7)

**Comment by gitbobobo** [comment]
https://devmusic.aqzscn.cn/docs/guides/operations
